### PR TITLE
Fixes space between `Search by Creator` and `Clear filters`

### LINKED ISSUE #799: Change the space between `Search by Creator` and `Clear filters`
In the search result page, the distance between filter `Search by Creator` and `Clear filters` is not correct. It should be `24px` with a line separating the actions.

### Expected design
Screen Shot 2020-03-24 at 17 09 38

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
Fixes #799 

clear

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: src/components/SearchGridFilter.vue [hunk 1]
```diff
@@ -131,8 +131,9 @@ export default {
 
 .search-filters_search-by,
 .clear-filters {
-  margin-top: 0.4em;
-  margin-left: 24px;
+  margin: 24px;
```
Comment (brenoferreira) [inline src/components/SearchGridFilter.vue]: You should use the Vocabulary spacing classes for margins and paddings https://cc-vocabulary.netlify.com/?path=/docs/tokens-spacing--figma

File: src/components/SearchGridFilter.vue [hunk 2]
```diff
@@ -131,8 +131,9 @@ export default {
...
+  border-bottom: 2px solid whitesmoke;
```
Comment (brenoferreira) [inline src/components/SearchGridFilter.vue]: Please remove this custom border style

File: src/components/SearchGridFilter.vue (hunk 1 above)
Comment (abhisheknaiidu) [reply src/components/SearchGridFilter.vue]: Yup Done :)

File: src/components/SearchGridFilter.vue (hunk 2 above)
Comment (abhisheknaiidu) [reply src/components/SearchGridFilter.vue]: Done :)

File: src/components/SearchGridFilter.vue [hunk 3]
```diff
@@ -112,6 +112,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import "@creativecommons/vocabulary/scss/spacing.scss";
```
Comment (brenoferreira) [inline src/components/SearchGridFilter.vue:115]: You don't have to import the .scss file to use the variables. You can use the `margin` and `padding` classes in the appropriate element. See https://cc-vocabulary.netlify.com/?path=/docs/tokens-spacing--figma#usage

Comment (abhisheknaiidu) [reply src/components/SearchGridFilter.vue]: Done @brenoferreira

**Comment by abhisheknaiidu** [comment]
@brenoferreira  looks like this now -

margin

**Pull request merged**